com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs

```diff
@@ -302,6 +302,10 @@ internal void SetParentCamera(HDCamera parentHdCam)
         // XR multipass and instanced views are supported (see XRSystem)
         internal XRPass xr { get; private set; }
 
+        internal float globalMipBias { set; get; } = 0.0f;
+
+        internal void SetGlobalMipBiasFromCamera() { globalMipBias = m_AdditionalCameraData == null ? 0.0f : m_AdditionalCameraData.materialMipBias; }
```

Comment:
is it valid to have m_AdditionalCameraData null in HDCamera? @RSlysz

Reply:
Hum HDCamera is a class that aggregate all data from Camera and Additional data to speed up frame computation in the pipeline.
I am not 100% sure here. 
@JulienIgnace-Unity  Any idea ?

Reply:
IMO it should have been deal with null before and if null we should have use the default additional data here.

Reply:
Removed the SetGlobalMipBiasFromCamera function, and instead moved all this to Update. We do have to do the null check there, because we could be dealing with a reflection probe camera (which might have additional camera as null)